command/events.go

```diff
@@ -88,6 +98,21 @@ func (c *EventsSubscribeCommands) Run(args []string) int {
 	return 0
 }
 
+// cleanNamespace removes leading and trailing space and /'s from the namespace path.
+func cleanNamespace(ns string) string {
+	ns = strings.TrimSpace(ns)
+	ns = strings.Trim(ns, "/")
+	return ns
+}
+
+func cleanNamespaces(namespaces []string) []string {
```

Comment:
I'm surprised this isn't a utility function somewhere in Vault.

Reply:
There are some methods in https://github.com/hashicorp/vault/blob/main/helper/namespace/namespace.go but they aren't quite what we are looking for. (They canonicalize by adding the `/` at the end, which I don't think is quite what we want?)

And removing spaces seems prudent given this is coming from user input.